B3-3iL-DLW/3il-student
# Refactor/refactor data layer api

- refactor all data layer entities/models/repositories
- created some temporary views to test

how to test :
- choose your page to test in the main_dev.dart 

the views are just to check if we can get all the datas
